src/main/scala/geotrellis/sdg/RenderPyramidApp.scala

```diff
@@ -0,0 +1,48 @@
+package geotrellis.sdg
+
+import com.monovore.decline.{CommandApp, Opts}
+import org.apache.spark._
+import org.apache.spark.sql._
+import java.net.URI
+import cats.implicits._
+import geotrellis.raster.RasterSource
+import geotrellis.layer.ZoomedLayoutScheme
+import geotrellis.spark._
+import geotrellis.proj4._
+import geotrellis.layer._
+import geotrellis.raster.resample.Bilinear
+
+object RenderPyramidApp extends CommandApp(
+  name = "RenderPyramidApp",
+  header = "Render GeoTiff(s) as pyramid",
+  main = {
+    val rasterOpts = Opts.options[URI](
+      long = "raster", short = "r",
+      help = "Input raster")
+
+    val tileLayerUriPrefixOpt = Opts.option[URI](
```

Comment:
For future programs, these two CLI arguments should probably be `Opts.argument` rather than `Opts.option` since they're required for the program to function and don't have sensible defaults. Just an advisory note and not expecting a fix so that these two CLIs are consistent, since I realize that over in #37 we're still using `Opts.option` for similar args and I haven't fixed it there.

Reply:
Thanks for the call-out, the distinction was lost on me until now.